ice.c

```diff
@@ -3617,7 +3617,21 @@ static gint rtcp_transport_wide_cc_stats_comparator(gconstpointer item1, gconstp
 static gboolean janus_ice_outgoing_transport_wide_cc_feedback(gpointer user_data) {
 	janus_ice_handle *handle = (janus_ice_handle *)user_data;
 	janus_ice_peerconnection *pc = handle->pc;
-	janus_ice_peerconnection_medium *medium = pc ? g_hash_table_lookup(pc->media_bytype, GINT_TO_POINTER(JANUS_MEDIA_VIDEO)) : NULL;
+
+	janus_ice_peerconnection_medium *medium;
+	if (pc) {
+		/* Find inbound video medium */
+		GHashTableIter iter;
+		gpointer value;
+		g_hash_table_iter_init(&iter, pc->media_bymid);
+		while (g_hash_table_iter_next(&iter, NULL, &value)) {
+			janus_ice_peerconnection_medium *m = value;
+			if (m->type == JANUS_MEDIA_VIDEO && m->recv) {
```

Comment:
Same as the `if` above. Besides, you should also check if `m` is a valid pointer: as it is, you're dereferencing the pointer right away, which would crash if there's no value.